app/assets/javascripts/src/Metamaps.Map.js

```diff
@@ -436,11 +436,25 @@ Metamaps.Map.InfoBox = {
     })
     $('body').click(self.close)
 
-    self.attachEventListeners()
-
-    
+    self.attachEventListeners() 
 
     self.generateBoxHTML = Hogan.compile($('#mapInfoBoxTemplate').html())
+    
+    var querystring = window.location.search.replace(/^\?/, '')
+    if (querystring == 'new') { 
+      self.open()
+      window.setTimeout(self.requestMapTitle, 3000)
+    }
+  },
+  requestMapTitle: function () {
+    var t = prompt('Please enter a name for your map. You can also set it any time using the map info panel.')
```

Comment:
Are prompts too 90's? Should we get some kind of modal dialog library, like bootbox or something?

I'm thinking this sentence could be better..... not exactly sure. Maybe

Please enter a name for your map. You can change this later at any time using the map info box.

Box/panel? Is it clear what that means? Before the map info box could be blissfully ignored if you were new to Metamaps, but now it's a new introduced concept. Dunno. How do people do those interactive tutorials on people's first sign ins?